rma_account/models/account_move.py

```diff
@@ -272,3 +269,40 @@ def _stock_account_get_anglo_saxon_price_unit(self):
                     self.company_id
                 )._compute_price(average_price_unit, self.product_uom_id)
         return price_unit
+
+    def _get_computed_name(self):
+        self.ensure_one()
+        if not self.product_id:
+            return ""
+        if self.partner_id.lang:
+            product = self.product_id.with_context(lang=self.partner_id.lang)
+        else:
+            product = self.product_id
+        values = []
+        if product.partner_ref:
+            values.append(product.partner_ref)
+        if self.journal_id.type == "sale":
+            if product.description_sale:
+                values.append(product.description_sale)
+        elif self.journal_id.type == "purchase":
+            if product.description_purchase:
+                values.append(product.description_purchase)
+        return "\n".join(values)
+
+    def _get_computed_account(self):
+        self.ensure_one()
```

Comment:
Same here, not sure it is needed, it is already computed here : https://github.com/odoo/odoo/blob/16.0/addons/account/models/account_move_line.py#L487

Reply:
Done!